db/seeds/002_recipes.rb

```diff
@@ -195,6 +209,7 @@
     description: "Kebabs are a quintessential summer recipe, and our version got a sweet upgrade with fresh pineapple, red bell pepper, and a mixture of honey, soy sauce, and apple cider vinegar. If you have a bit of extra time, you could marinate the kebabs first, but it's equally delicious brushed on while grilling.",
     notes: "If using wooden skewers, soak them in water for 30 minutes before threading to avoid them charring or catching on fire.\n\nTry using steak instead of  pork and mango instead of pineapple.",
     category_names: %w[Pork Main],
+    user_id: 1
```

Comment:
Not a fan of using hardcoded ids. You should find the existing user instead. Either User.first or find by email. Also, since all recipes are getting the same user, you can just set it on the loop below

user.recipes.find_or_initialize...